src/webgldrawer.js

```diff
@@ -870,9 +865,14 @@
             let position;
 
             let data = cache.data;
+            let canvas = false;
+            if (data instanceof CanvasRenderingContext2D) {
```

Comment:
This was a bug, context reference must be converted to the canvas one. Question is, how we discover that data is tainted if the data is not a canvas reference..? E.g. with a blob.